2-projects/1-todo-list/script.js

```diff
@@ -9,17 +38,40 @@ let todos = [
   { task: "Wash the dishes", completed: false },
   { task: "Do the shopping", completed: false },
 ];
+console.log(todos);
+let form = document.getElementById("todo-form");
+form.addEventListener("submit", addNewTodo);
 
-populateTodoList(todos);
-
-// This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
-function addNewTodo(event) {
-  // The code below prevents the page from refreshing when we click the 'Add Todo' button.
-  event.preventDefault();
-  // Write your code here... and remember to reset the input field to be blank after creating a todo!
+function addNewTodo(e) {
+  e.preventDefault();
+  if (!input.value) {
+    return;
+  } else {
+    let obj = Object.assign({
+      task: input.value,
+      completed: window.getComputedStyle(e.target).textDecoration
+        ? true
+        : false,
+    });
+    todos.push(obj);
+  }
+  populateTodoList(input.value);
```

Comment:
You should create an object here with "task" and "completed" properties, like to prepopulated values.